### ISSUE TITLE
Drop WindowsLive provider

### ISSUE THREAD

**Issue by chrisgraham** [hybridauth/hybridauth#1210]
From what I can see, the WindowsLive OAuth2 API is deprecated.
The documentation at https://msdn.microsoft.com/en-us/library/hh243647.aspx no longer exists.
I can't find any alternative documentation.
I don't know where I could set up an API key to test.

It seems to me MicrosoftGraph works as the current recommended way to do it, which we support well.

That said the WindowsLive one does return DOB, profile URLs, gender.

Maybe put this down for deletion in the next major update?

**Comment by nightw0lv** [comment]
yes indeed i am using microsoft graph too windowslive should be removed even the application is not supported anymore

**Comment by ApacheEx** [comment]
I agree. Let's plan to remove it in v4.

**Comment by AdamchukAndrii** [comment]
Hello @ApacheEx can you tell when you are planning to run v4?

**Comment by ApacheEx** [comment]
well I'm not sure, it depends on the community since I don't have a time at this moment. I've feeling it would be possible at the end of this year.
